test/clojure-mode-font-lock-test.el

```diff
@@ -101,7 +101,13 @@ POS."
 
 (ert-deftest clojure-mode-syntax-table/fontify-clojure-keyword ()
   :tags '(fontification syntax-table)
-  (should (eq (clojure-test-face-at 2 11 "{:something 20}") 'clojure-keyword-face)))
+  (should (equal (clojure-test-face-at 2 11 "{:something 20}") '(clojure-keyword-face))))
+
+(ert-deftest clojure-mode-syntax-table/fontify-namespaced-keyword ()
+  :tags '(fontification syntax-table)
+  (should (equal (clojure-test-face-at 2 2 "{:alias/some 20}")  '(clojure-keyword-face)))
```

Comment:
I agree with @expez 100%. The change is for the better and is not going anywhere.